app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_save :update_post_comment
+  def update_post_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ ] Well done using `after_save ` to call the `#update_post_counter ` method every time a new post has been saved to database, I  just recommend to make the update_post_counter method private.


`after_save :update_post_comment

private

  def update_post_counter
    author.increment!(:posts_counter)
  end`